README.md

```diff
@@ -1,3 +1,40 @@
-# react-native-button
+# Native Button
+## A native button for React.
+[![Gitter](https://badges.gitter.im/Devnetik/react-native-button.svg)](https://gitter.im/Devnetik/react-native-button?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Like i said in the chat just rename every react-native-button to react-native-easy-button and this pr is good.